aiohttp/web.py

```diff
@@ -421,7 +421,8 @@ def run_app(app, *, host=None, port=None, path=None, sock=None,
         loop.run_until_complete(app.shutdown())
         loop.run_until_complete(handler.shutdown(shutdown_timeout))
         loop.run_until_complete(app.cleanup())
-    loop.close()
+    if close_loop:
```

Comment:
I think this is very good idea. instead of extra parameter, let's just close owned loop. if it come from outside just do nothing.